integrations/getService/index.test.ts

```diff
@@ -0,0 +1,86 @@
+import { getEnvValue } from "../../lib/env";
+import fetchApi from "../../lib/fetch";
+
+import { basicResponseDecoder } from "italia-ts-commons/lib/requests";
+import { ServicePublic } from "../../generated/definitions/backend/ServicePublic";
+
+const decoder = basicResponseDecoder(ServicePublic);
+
+describe("getService", () => {
+  const host = getEnvValue("IO_BACKEND_HOST").getOrElseL(() => {
+    throw new Error(`required value dor "IO_BACKEND_HOST"`);
+  });
+  const basePath = getEnvValue("IO_BACKEND_BASEPATH").getOrElseL(() => {
+    throw new Error(`required value dor "IO_BACKEND_BASEPATH"`);
+  });
+  const sessionToken = getEnvValue("SPID_SESSION_TOKEN").getOrElseL(() => {
+    throw new Error(`required value dor "SPID_SESSION_TOKEN"`);
+  });
+  const fakeServiceId = "fake_id";
+  const existingServiceId = "io-dev-azure";
```

Comment:
I hate this, but at the moment we have no other way to specify an existing service.

We could instead write a _scenario_  which fetches the list first, and them retrieve the single detail. But it seems a bit overkill to me.

Reply:
that's right, just use a different naming for mock objects and var:

`aFakeServiceId`
`anExistingServiceId`

so we can recognize it